versioned_docs/version-Release/alert-subscribe/easy-subscribe/index.md

```diff
@@ -0,0 +1,110 @@
+
+# Easy Subscribe
+With the **Easy Subscription** feature, users can subscribe to Community Manager topics directly, without needing to create a Notifi account, sign up on the Notifi React Card, manage preferences, or sign their wallet. Users can subscribe using their email and can opt to subscribe to the Telegram bot.
+
+# Creating an Easy Subscribe Card Config
+
+To use the **Easy Subscribe** feature, a card configuration must be created. This configuration sets up Easy Subscription campaign and provides the necessary endpoints, allowing you to directly use these URLs to join the Telegram bot and send email addresses for subscribing to Community Manager topics.
```

Comment:
fix: a respective Easy Subscribe card configuration must be created.
fix: sets up an Easy Subscription campaign

fix: necessary endpoints: allowing you to directly.......

fix: and send to email addresses